plugin.video.4players/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.4players" name="4Players Videos" version="2.0.3" provider-name="Tristan Fischer (sphere@dersphere.de)">
+<addon id="plugin.video.4players" name="4Players Videos" version="2.0.4" provider-name="Tristan Fischer (sphere@dersphere.de)">
     <requires>
         <import addon="xbmc.python" version="2.1.0"/>
```

Comment:
Not sure if @MartijnKaijser comment was about the python API version or the overall PR. If not the second please merge it.